lib/ukring/include/uk/ring.h

```diff
@@ -334,6 +334,8 @@ uk_ring_peek_clear_sc(struct uk_ring *br)
 	 */
 	#error "unsupported: atomic_thread_fence_acq()"
 	/* TODO atomic_thread_fence_acq(); */
+#elif defined(CONFIG_ARCH_ARM64)
+	ukarch_thread_fence_acq();
```

Comment:
From what I see the name of this function was taken from the comment right above, yet I'm not very keen of it as it looks like it originates from [atomic_thread_fence of C++](https://en.cppreference.com/w/cpp/atomic/atomic_thread_fence) and we don't have a `ukarch_thread` API. I think a simple `dmb()` will do instead.

Reply:
fixed